hw/ip/alert_handler/doc/_index.md

```diff
@@ -902,32 +902,28 @@ not be triggered by this testing mechanism.
 
 ## Register Table
 
-The below register description can be generated with the `reg_alert_handler.py`
-script. The reason for having yet another script for register generation is
-that the alert handler is configurable for the number of alert sources
-(similar to the rv_plic design).
+The register description below matches the instance in Top Earlgrey.
```

Comment:
"Top Earlgrey" -> "earlgrey top level design" (or Earl Grey, we don't really have a consistent naming convention)

Reply:
Done